drivers/watchdog/Kconfig.esp32

```diff
@@ -8,14 +8,23 @@
 menuconfig WDT_ESP32
 	bool "ESP32 Watchdog (WDT) Driver"
 	depends on SOC_ESP32
+	select HAS_DTS_WDT
 	default y
 	help
 	  Enable WDT driver for ESP32.
 
-config WDT_ESP32_IRQ
+config WDT0_ESP32_IRQ
```

Comment:
With DTS, you can use the config coming from the DTS, and there is no need for these IRQ kconfig anymore.

Reply:
Ah... I thought DTS #define would still be there, but without the parent interrupt controller, the interrupt property cannot be set.